src/utils/object.ts

```diff
@@ -0,0 +1,27 @@
+/********************************************************************************
+ * Copyright (c) 2017-2021 TypeFox and others.
```

Comment:
Please use the current year for new files, and change the affiliation to your own if you like.